library/src/main/java/com/datatorrent/lib/parser/Parser.java

```diff
@@ -61,25 +70,48 @@ public void setup(PortContext context)
     }
   };
 
-  @OutputPortFieldAnnotation(optional = true)
   public transient DefaultOutputPort<INPUT> err = new DefaultOutputPort<INPUT>();
```

Comment:
@ishark please consider my comment as just a suggestion. I don't have a strong opinion here. These things evolve as we see more use cases.